tests/unit/test_tiler.py

```diff
@@ -152,6 +152,9 @@ def it_can_generate_random_coordinates(self, request, tmpdir):
 
         _box_mask_thumb.assert_called_once_with(binary_mask, slide)
         _tile_size.assert_has_calls([call((128, 128))])
+        _random_choice_true_mask2d.assert_called_once_with(
+            NpArrayMock.ONES_500X500_BOOL
+        )
         _scale_coordinates.assert_called_once_with(
             reference_coords=CP(x_ul=0, y_ul=0, x_br=128, y_br=128),
             reference_size=(500, 500),
```

Comment:
i don't understand why this is not enough? 
this
```
_random_choice_true_mask2d.return_value = (0, 0)
```
is basically asserting that x and y are 0 and so this line asserts the same thing i believe

Reply:
here I assert the fact that starting from (0,0) - which I know because I mocked `random_choice_true_mask2d` - I get to (128,128) via the scaling of the tile dimensions and then the sum.. but I'm not really testing if the coordinates UL are correct

Reply:
oh i see they're random that's the point. I don't know maybe the integration tests are enough to cover this .... but if you feel more safe this way, i can buy that. So... just let me know and I'll stamp the PR

Reply:
I would prefer to keep the unit test as it is, via the integration tests we have we never noticed (visually), probably due to a lucky seed

Reply:
aight